creator-node/src/services/sync/syncImmediateQueue.js

```diff
@@ -88,6 +88,23 @@ class SyncImmediateQueue {
     if (prometheusRegistry !== null && prometheusRegistry !== undefined) {
       prometheusRegistry.startQueueMetrics(this.queue, worker)
     }
+
+    // any leftover active jobs need to be deleted when a new queue
+    // is created since they'll never get processed
+
+    // eslint-disable-next-line @typescript-eslint/no-floating-promises
+    this.deleteOldActiveJobs()
```

Comment:
thanks for fixing - can you move this to before the worker creation to avoid race conditions? probably nbd but I want to make sure it doesn't start processing or adding some other jobs and then remove them

Reply:
Makes sense, I'll move it